### PR TITLE
CLOUDP-93385: Retrieves all unaccepted invitations to the specified Atlas organization

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-93385

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli iam organizations invitations list
ID                         USERNAME                      CREATED AT             EXPIRES AT
60c884d12dbba167ddbe650c   test@mongodb.com   2021-06-15T10:45:37Z   2021-07-15T10:45:37Z
60d9d7509b5dce4c7b050c35  test@gmail.com   2021-06-28T14:06:08Z   2021-07-28T14:06:08Z
```

```
 ./bin/mongocli iam organizations invitations list -o json
[
  {
    "id": "60c884d12dbba167ddbe650c",
    "orgId": "5e429e7706822c6eac4d5970",
    "orgName": "CLI Atlas",
    "createdAt": "2021-06-15T10:45:37Z",
    "expiresAt": "2021-07-15T10:45:37Z",
    "inviterUsername": "test@mongodb.com",
    "username": "test@mongodb.com",
    "roles": [
      "GROUP_OWNER"
    ]
  },
  {
    "id": "60d9d7509b5dce4c7b050c35",
    "orgId": "5e429e7706822c6eac4d5970",
    "orgName": "CLI Atlas",
    "createdAt": "2021-06-28T14:06:08Z",
    "expiresAt": "2021-07-28T14:06:08Z",
    "inviterUsername": "test@mongodb.com",
    "username": "test@gmail.com",
    "roles": [
      "ORG_MEMBER"
    ]
  }
]
```

### CODE REVIEW

File: internal/cli/iam/organizations/invitations/invitations.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "invitations"
+	cmd := &cobra.Command{
+		Use:     use,
+		Short:   "Invitation operations.",
+		Long:    "Create, list and manage your MongoDB organization invites.",
+		Aliases: cli.GenerateAliases(use, "orgs", "org"),
```
Comment (gssbzn) [inline internal/cli/iam/organizations/invitations/invitations.go:27]: 👀  copy/paste?

Comment (andreaangiolillo) [reply internal/cli/iam/organizations/invitations/invitations.go:27]: yes 😿

File: internal/cli/iam/organizations/invitations/list_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+// +build unit
+
+package invitations
+
+import (
+	"testing"
+
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/test"
+
```
Comment (gssbzn) [inline internal/cli/iam/organizations/invitations/list_test.go:24]: please delete the extra empty line

File: internal/cli/iam/organizations/organizations_test.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 MongoDB Inc
```
Comment (gssbzn) [inline internal/cli/iam/organizations/organizations_test.go:1]:
```suggestion
// Copyright 2021 MongoDB Inc
```

**Review by gssbzn — CHANGES_REQUESTED** [review]
goos start, main thing the aliases for invitations are wrong

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**